scripts/fill_help_tables.sql

```diff
@@ -1,13 +1,5 @@
-<<<<<<< HEAD
```

Comment:
How did this happen that this file has a merge conflict?
As far as I understood it should have been just overwritten in the `(docs)` commit.

Reply:
Good call. I didn't notice. I have done some investigation and it seems it is because the file has it have been change in a commit in a way that it is different from the one in tarball.

- This is the commit: <https://github.com/mysql/mysql-server/blob/a2036369538dc1cca9d2f7fad817d79edb27cead/scripts/fill_help_tables.sql#L1>
- This is the one from the tarballs: <https://github.com/ldonoso/percona-server/blob/33bcc3e09787b21d4b84adbe94bea11e59a836a9/scripts/fill_help_tables.sql#L1>

Because of this, I realize that maybe the current changes should have prevailed, @percona-ysorokin , please, what do you think?

Reply:
@ldonoso In git, in 5.7 branch Oracle has a dummy placeholder for this file
https://github.com/mysql/mysql-server/blob/5.7/scripts/fill_help_tables.sql
And it looks like in 8.0.24 it became a victim of mass copyright notice update :)
https://github.com/mysql/mysql-server/commit/a2036369538dc1cca9d2f7fad817d79edb27cead#diff-d5f778dc4ed23daf1900af734508992217797aa7a481c561f81017394bef7ce8
So, the correct way to resolve the conflict would be to take the version from the `(doc)` commit.

Reply:
Many thanks for the investigation. I will fix it.